spec/check_spec.rb

```diff
@@ -4,19 +4,12 @@
   describe "check" do
     context "given 2 dupes" do
       it "returns 2" do
-        dupes = %(
-          https://github.com
-          https://github.com
-        )
+        dupes = %(https://github.com https://github.com)
```

Comment:
The markdown parser parses the leading whitespace as a code block:

          https://github.com
          https://github.com

https://github.com https://github.com